add working with assessment doc (#49611)
dotnet/docs: docs/azure/migration/appmod/working-with-assessment.md

---
title: "Working with Assessment: Comprehensive guide to application assessment"
description: Learn how to effectively work with application assessments in GitHub Copilot app modernization, including configuration, interpretation, and report management.
author: KarlErickson
ms.author: karler
ms.reviewer: fenzho
ms.topic: concept-article
ms.date: 11/04/2025
ms.custom: devx-track-dotnet
ai-usage: ai-assisted
---

> # Working with Assessment: Comprehensive guide to application assessment

Reviewer: ```suggestion
# Application assessment with GitHub Copilot app modernization
```

| Target                            | Description                                                        |
|-----------------------------------|--------------------------------------------------------------------|
| Any                               | Discover issues for all supported targets here.                    |
| AKS.Windows                       | Best practices for Azure Kubernetes Service (Windows).             |
| AKS.Linux                         | Best practices for Azure Kubernetes Service (Linux).               |
| AppService.Windows                | Best practices for Azure App Service (Windows).                    |
| AppService.Linux                  | Best practices for Azure App Service (Linux).                      |
| AppServiceContainer.Windows       | Best practices for Azure App Service Container (Windows).          |
| AppServiceContainer.Linux         | Best practices for Azure App Service Container (Linux).            |
| AppServiceManagedInstance.Windows | Best practices for Azure App Service Managed Instance (Windows).   |
| ACA                               | Best practices for Azure Container Apps.                           |

### Examples

> The following are two examples of how to configure properly.

Reviewer: ```suggestion
The following provide two examples of proper configurations:
```

- **Issue Summary**: Overview of migration issues categorized by domain with criticality percentages.
- **Issues**: Provides a concise summary of all issues that require attention.

:::image type="content" source="./media/assessment-report-dashboard.png" lightbox="./media/assessment-report-dashboard.png" alt-text="Screenshot of Visual Studio showing the GitHub Copilot app modernization assessment report dashboard.":::

#### Issues

The issues section provides a categorized list of various aspects of Cloud Readiness that you need to address to successfully migrate the application to Azure. The following tables describe the `Domain` and `Criticality` values:

| Domain              | Description                                                                             |
|---------------------|-----------------------------------------------------------------------------------------|
| **Cloud Readiness** | Evaluates app dependencies to suggest Azure services and ensure cloud-native readiness. |

| Criticality         | Description                                                   |
|---------------------|---------------------------------------------------------------|

> | **Mandatory**       | Issues that must be fixed for migration to Azure.             |

Reviewer 2: Do we reveal informational level as well? if so we need add it here.

Author: no, we don't
